README.md

```diff
@@ -138,19 +111,21 @@ bindings on your own.
    necessary submodules, build `play-cpp-sdk` crate, finally setup and build the demo project.
 2. Clean `~/.cargo/git/checkouts` if cxx fails to build, then run `windows_build.bat` again.
 3. Run `windows_install.bat`, libraries and bindings will be copied into a new created folder:
-   `build`
+   `install`
 
 ### Notes about Visual Studio 2022
 1. Open `demo.sln`. If you use Visual Studio 2022, retarget project, and upgrade
    PlatformToolset to `v143` before running `windows_build.bat`
 
 ## Mac
-1. Run `make build_cpp` or `make cppx86_64`
-2. Run `make install`, libraries and bindings will be copied into a new created folder: `build`
+1. Run `make`
+2. Run `make install`, libraries and bindings will be copied into a new created folder: `install`
 
 ### Linux
-1. Run `make build_cpp`
-2. Run `make install`, libraries and bindings will be copied into a new created folder: `build`
+1. Install development packages of openssl, for example, `libssl-dev` on Ubuntu or
+   `openssl-devel` on Fedora.
```

Comment:
there was this fix: https://github.com/crypto-com/defi-wallet-core-rs/pull/478 I wonder if one could go without openssl?

Reply:
let me test

Reply:
Done